cmake/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if (CYGWIN)
 else()
```

Comment:
Why doesn't the if statement above take care of the case where we need -std=gnu++11?